src/routes/__tests__/browse.spec.ts

```diff
@@ -24,7 +24,7 @@ describe("addItemLinkIfPresent", () => {
 
     const numberoOfLinks = stacCollection.links.length;
     // Invoke method
-    addItemLinkIfPresent(stacCollection, "https://foo.com/items");
```

Comment:
We were talking about this on slack and I'm not sure I understand. Is it the case that this part of the url 'items' is going to be in the actual metadata and thus if we append it going to cause a problem?

Reply:
In the case of CMR collections with granules, the metadata will not contain a URL for the items endpoint, it is inferred and generated in code. CMR collections without granules and a STAC related url will have their link pulled from metadata.